packages/kbn-cases-components/src/tooltip/translations.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const OPENED = i18n.translate('cases.components.tooltip.opened', {
+  defaultMessage: 'Opened',
+});
+
+export const CLOSED = i18n.translate('cases.components.tooltip.closed', {
+  defaultMessage: 'Closed ',
```

Comment:
Any reason why there is a whitespace char at the end of `CLOSED` but not for `OPENED`?

Reply:
Not really, it was before adding `{' '}`. it is not needed anymore. I will remove it